frontend/ui/widget/bookmapwidget.lua

```diff
@@ -1269,7 +1274,7 @@ function BookMapWidget:showMenu()
         },
         not self.overview_mode and {
             {
-                text = _("Page slot width"),
+                text = _("Bar width"),
```

Comment:
What's the problem with "Page slot" ?
"Bar width" doesn't tell much what bar it is.
"Page bar width" ? if "slot" stinks?
And in effect, a page slot/room often has a bar - unless it has not been read yet.